KarhooUISDK/Workers/KarhooMailMetaInfoComposer.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  KarhooMailMetaInfoComposer.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 02/02/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+protocol KarhooMailMetaInfoComposerProtocol {
```

Comment:
`Karhoo` as a prefix is reserved for types that implement a protocol and should be avoided in protocol names. Maybe rename the protocol to `MailMetaInfoComposer` and the class to `KarhooMailMetaInfoComposer`?